packages/replay/src/canvas.ts

```diff
@@ -19,16 +28,20 @@ export class ReplayCanvas implements Integration {
    */
   public name: string;
 
+  public canvasManager: (options: GetCanvasManagerOptions) => CanvasManagerInterface;
```

Comment:
So I think this will be a bit more straight forward if `canvasManager` is the instance of `CanvasManager` instead of a function that returns one. This way we have a single instance of `CanvasManager` instead of creating a new one for every snapshot.

This means that we will need to create a new public function in rrweb e.g. `createCanvasManager` that accepts an options object and returns a new instance of `CanvasManager`.